inspektor/check.py

```diff
@@ -120,21 +120,17 @@ def check_file(self, path):
 
 class PatchChecker(FileChecker):
 
-    def __init__(self, args, patch=None, patchwork_id=None, github_id=None,
+    def __init__(self, args, patch=None,
```

Comment:
I'm wondering whether you want to create a utility, or abstract interface. If the later you should probably add `validate`, `check` and `check_modified_files` to the `FileChecker` raising `NotImplementedError`.